scripts/remove-redundant-flags.js

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const { platform } = require('os');
+
+const compareVersions = require('compare-versions');
+
+const bcd = require('..');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
+
+let twoYearsAgo;
+if (
+  process.env.npm_lifecycle_event &&
+  process.env.npm_lifecycle_event.includes('mocha')
+) {
+  // When running test, use a specific time for consistent testing
+  twoYearsAgo = new Date('January 1, 2019 0:00:00');
+} else {
+  twoYearsAgo = new Date(Date.now());
+  twoYearsAgo.setFullYear(twoYearsAgo.getFullYear() - 2);
+}
+
+const getEarliestVersion = (...args) => {
+  const versions = args
+    .filter(version => typeof version === 'string')
+    .map(version => version.replace('≤', ''));
+
+  let earliestVersion = versions[0];
+
+  for (const version of versions) {
+    if (
+      earliestVersion === 'preview' ||
+      (version !== 'preview' &&
+        compareVersions.compare(earliestVersion, version, '>'))
+    )
+      earliestVersion = version;
+  }
+
+  return earliestVersion;
+};
+
+const removeRedundantFlags = (key, value, limitBrowser) => {
+  if (key === '__compat') {
+    for (const [browser, rawSupportData] of Object.entries(value.support)) {
+      if (limitBrowser && browser != limitBrowser) continue;
+
+      if (rawSupportData !== undefined) {
```

Comment:
Is it really possible for `rawSupportData` to be undefined here? (I don't think so.)